tests/test_core.py

```diff
@@ -138,13 +138,21 @@ def test_validate_up(meta_df):
     assert len(obs) == 1
     assert obs['year'].values[0] == 2010
 
+    assert list(meta_df['exclude']) == [False, False] # assert none excluded
+
 
 def test_validate_lo(meta_df):
-    obs = meta_df.validate({'Primary Energy': {'lo': 2.0}}, exclude=False)
+    obs = meta_df.validate({'Primary Energy': {'up': 8, 'lo': 2.0}})
     assert len(obs) == 1
     assert obs['year'].values[0] == 2005
```

Comment:
Last comment here, and I think it will be quick to implement. I believe that `a_scenario` is the one being caught here, correct? Could you test the scenario too (for clarity for potential future tests)?